build.sbt

```diff
@@ -25,7 +25,7 @@ lazy val commonSettings = Seq(
     Wart.OptionPartial),
   organization := "io.horizen",
   organizationName := "Zen Blockchain Foundation",
-  version := "2.0.0-RC10-SNAPSHOT",
+  version := "2.0.0-RC10-SNAPSHOT2",
```

Comment:
I know it's not related, but just not to create another PR, can you please update on line 60
`"commons-net" % "commons-net" % "3.8.0"`
->
`"commons-net" % "commons-net" % "3.9.0"`
it fixes a vulnerability
Thanks!